src/System.Data.Odbc/src/System/Data/Odbc/OdbcParameter.cs

```diff
@@ -346,7 +346,7 @@ private int GetColumnSize(object value, int offset, int ordinal)
             {
                 if (ODBC32.SQL_C.NUMERIC == _typemap._sql_c)
                 {
-                    cch = 62;  // (DecimalMaxPrecision+sign+terminator)*BytesPerUnicodeCharater
+                    cch = 62;  // (DecimalMaxPrecision+sign+terminator)*BytesPerUnicodeCharacter
```

Comment:
Where is BytesPerUnicodeCharacter defined, and how does this compile if you only change the usage site and not the declaration?